pcdsdevices/slits.py

```diff
@@ -602,6 +602,18 @@ def calc_lightpath_state(
                                             ywidth=ywidth_readback)
 
 
+class SL2K4Slits(BeckhoffSlits):
+    def __init__(self, prefix, cam, data_source, *, name, **kwargs):
+        self._started_move = False
+        self._top_done = False
+        self._bottom_done = False
+        self._north_done = False
+        self._south_done = False
+        self._cam = cam
+        self._data_source = data_source
```

Comment:
Suggestion: this class needs some brief documentation about what `cam` and `data_source` are, how one would get them, and what we use them for.

Reply:
I don't immediately see where they are used, but I was more concerned about documenting what they are for our fellow team members who see this in the future.